src/lambda/wizards/samDeployWizard.ts

```diff
@@ -697,10 +698,15 @@ export class SamDeployWizard extends MultiStepWizard<SamDeployWizardResponse> {
 
             try{
                 const s3Client = ext.toolkitClientBuilder.createS3Client(this.response.region!)
-                this.response.s3Bucket = (await s3Client.createBucket({bucketName: newBucketRequest})).bucket.name
+                const newBucketName = (await s3Client.createBucket({bucketName: newBucketRequest})).bucket.name
+                this.response.s3Bucket = newBucketName
+                getLogger().info('Successfully created bucket %O', newBucketName)
```

Comment:
- Never need to mention "success", it is implied.
- variable parts of the log message should live at the end following a `:`, or else in quotation marks.

```suggestion
                getLogger().info('Created bucket: %O', newBucketName)
```